### ISSUE TITLE
Saturation option 

### ISSUE THREAD

**Issue by engrchachi** [krojew/OpenRGB-Ambient#13]
colors not saturated specially violet which outputs white light

**Comment by krojew** [comment]
Can you explain more what's the issue?

**Comment by engrchachi** [comment]
if the colors on the monitor is light violet  or any light colors with the shade of red, the led light will output white color insted of the actual color. can be fixed if theres an option to change color saturation.

**Comment by krojew** [comment]
Did you try turning off cool led compensation or matching the color temperature to your monitor?

**Comment by engrchachi** [comment]
yes. already tried tweaking configurations available for this plugin.
